pwa/src/templates/templateParts/filters/FiltersTemplate.tsx

```diff
@@ -91,6 +105,20 @@ export const FiltersTemplate: React.FC<FiltersTemplateProps> = ({ isLoading }) =
     navigate(`/${filtersToUrlQueryParams(filters)}`);
   }, [filters]);
 
+  const getItems = useFilterCount().getCategoryCount();
```

Comment:
I don't think `getItems` is a descriptive name here, `getCategoryCount` might've been a better name. But with the other feedback implemented, `getCategories` might be best.